src/test/regress/expected/multi_transaction_recovery_multiple_databases.out

```diff
@@ -0,0 +1,360 @@
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+\c db1 - - :worker_1_port
+CREATE EXTENSION citus;
+\c db1 - - :worker_2_port
+CREATE EXTENSION citus;
+\c db1 - - :master_port
+CREATE EXTENSION citus;
+SELECT citus_add_node('localhost', :worker_1_port);
+ citus_add_node
+---------------------------------------------------------------------
+              1
+(1 row)
+
+SELECT citus_add_node('localhost', :worker_2_port);
+ citus_add_node
+---------------------------------------------------------------------
+              2
+(1 row)
+
+SELECT current_database();
+ current_database
+---------------------------------------------------------------------
+ db1
+(1 row)
+
+SELECT pg_sleep_for('5 SECONDS');
+ pg_sleep_for
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT *
+FROM pg_dist_node;
+ nodeid | groupid | nodename  | nodeport | noderack | hasmetadata | isactive | noderole | nodecluster | metadatasynced | shouldhaveshards
+---------------------------------------------------------------------
+      1 |       1 | localhost |    57637 | default  | t           | t        | primary  | default     | t              | t
+      2 |       2 | localhost |    57638 | default  | t           | t        | primary  | default     | t              | t
+(2 rows)
+
+CREATE DATABASE db2;
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+SELECT oid AS db2_oid
+FROM pg_database
+WHERE datname = 'db2'
+\gset
+\c - - - :worker_1_port
+CREATE DATABASE db2;
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+\c - - - :worker_2_port
+CREATE DATABASE db2;
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+\c db2 - - :worker_1_port
+CREATE EXTENSION citus;
+\c db2 - - :worker_2_port
+CREATE EXTENSION citus;
+\c db2 - - :master_port
+CREATE EXTENSION citus;
+SELECT citus_add_node('localhost', :worker_1_port);
+ citus_add_node
+---------------------------------------------------------------------
+              1
+(1 row)
+
+SELECT citus_add_node('localhost', :worker_2_port);
+ citus_add_node
+---------------------------------------------------------------------
+              2
+(1 row)
+
+SELECT current_database();
+ current_database
+---------------------------------------------------------------------
+ db2
+(1 row)
+
+SELECT pg_sleep_for('5 SECONDS');
```

Comment:
I assume this is to wait for the maintenance daemon to start. In other test files (multi_extension.sql) we used the following:

```
CREATE OR REPLACE FUNCTION test.maintenance_worker()
    RETURNS pg_stat_activity
    LANGUAGE plpgsql
AS $$
DECLARE
   activity record;
BEGIN
    DO 'BEGIN END'; -- Force maintenance daemon to start
    -- we don't want to wait forever; loop will exit after 20 seconds
    FOR i IN 1 .. 200 LOOP
        PERFORM pg_stat_clear_snapshot();
        SELECT * INTO activity FROM pg_stat_activity
        WHERE application_name = 'Citus Maintenance Daemon' AND datname = current_database();
        IF activity.pid IS NOT NULL THEN
            RETURN activity;   
        ELSE
            PERFORM pg_sleep(0.1);
        END IF ;
    END LOOP;
    -- fail if we reach the end of this loop
    raise 'Waited too long for maintenance daemon to start';
END; 
$$;
```
would be nice to do something similar here to avoid unnecessarily slow tests

Reply:
Done